sdk/storage/azure-storage-files-shares/swagger/README.md

```diff
@@ -590,6 +591,8 @@ directive:
       $.HandleItem.properties["LastReconnectTime"]["x-ms-client-name"] = "LastReconnectedOn";
       $.HandleItem.properties["Path"]["description"] = "File or directory name including full path starting from share root.";
       $.HandleItem["x-namespace"] = "_detail";
+      delete $.ListHandlesResponse.properties.HandleList["xml"];
+      $.ListHandlesResponse.properties.HandleList["x-ms-xml"] = {"name": "Entries", "wrapped": true};
```

Comment:
Update Changelog for this bug fix

Reply:
OK, I will update changelog in May Release PR.